Resources/template.json

```diff
@@ -0,0 +1,297 @@
+                  "name": "AzureWebJobsStorage",
+                  "value": "[parameters('storageConnectionString')]"
+                },
+                {
+                  "name": "CosmosDbSettings__EndpointUri",
+                  "value": "[parameters('cosmosDBfqdn')]"
+                },
+                {
+                  "name": "CosmosDbSettings__PrimaryKey",
+                  "value": "[parameters('cosmosDBkey')]"
+                },
+                {
+                  "name": "CosmosDbSettings__DatabaseId",
+                  "value": "[parameters('cosmosDBdatabase')]"
+                },
+                {
+                  "name": "CosmosDbCollectionSettings__FeChoicesCollectionId",
+                  "value": "[parameters('cosmosDbCollectionFeChoices')]"
+                },
+                {
+                  "name": "CosmosDbCollectionSettings__FrameworksCollectionId",
+                  "value": "[parameters('cosmosDbCollectionFrameworks')]"
+                },
+                {
+                  "name": "CosmosDbCollectionSettings__ProgTypesCollectionId",
+                  "value": "[parameters('cosmosDbCollectionProgTypes')]"
+                },
+                {
+                  "name": "CosmosDbCollectionSettings__SectorSubjectAreaTier1sCollectionId",
+                  "value": "[parameters('cosmosDbCollectionSsaTier1s')]"
+                },
+                {
+                  "name": "CosmosDbCollectionSettings__SectorSubjectAreaTier2sCollectionId",
+                  "value": "[parameters('cosmosDbCollectionSsaTier2s')]"
+                },
+                {
+                  "name": "CosmosDbCollectionSettings__StandardsCollectionId",
+                  "value": "[parameters('cosmosDbCollectionStandards')]"
+                },
+                {
+                  "name": "CosmosDbCollectionSettings__StandardSectorCodesCollectionId",
+                  "value": "[parameters('cosmosDbCollectionStandardsSectorCodes')]"
+                }
+              ]
+            },
+            "deployStagingSlot": {
+              "value": false
+            }
+          }
+        }
+      },
+      {
+        "name": "pprefdataApimProduct",
+        "type": "Microsoft.Resources/deployments",
+        "apiVersion": "2017-05-10",
+        "resourceGroup": "[parameters('ApimResourceGroup')]",
+        "properties": {
+          "mode": "Incremental",
+          "templateLink": {
+            "uri": "[concat(variables('deploymentUrlBase'), 'APIM/apim-product.json')]",
+            "contentVersion": "1.0.0.0"
+          },
+          "parameters": {
+            "allowAnonymousAccess": {
+              "value": true
+            },
+            "apimServiceName": {
+              "value": "[parameters('ApimServiceName')]"
+            },
+            "productDisplayName": {
+              "value": "Reference Data"
+            }
+          }
+        }
+      },
+      {
+        "name": "pprefdataApimApi",
```

Comment:
Looks like this was copy and pasted from another project without updating to reflect the name of this project